shared/src/androidTest/kotlin/tech.takahana.iconwallpaper.shared/BitmapImageAssetTest.kt

```diff
@@ -0,0 +1,32 @@
+package tech.takahana.iconwallpaper.shared
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+import tech.takahana.iconwallpaper.shared.assets.BitmapImageAsset
+import tech.takahana.iconwallpaper.shared.domain.domainobject.AssetId
+import tech.takahana.iconwallpaper.shared.domain.domainobject.AssetName
+
+class BitmapImageAssetTest {
+
+    private lateinit var bitmapImageAsset: BitmapImageAsset
+
+    @Before
+    fun before() {
+        bitmapImageAsset = BitmapImageAsset(
+            id = AssetId("assetId"), name = AssetName("assetName"), bitmap = mockk()
+        )
+    }
+
+    @Test
+    fun recycleBitmap() {
+        val bitmap = bitmapImageAsset.bitmap
+        every { bitmap.recycle() } returns Unit
+
+        bitmap.recycle()
```

Comment:
テスト対象クラスは `bitmapImageAsset` なので以下のようになりそうです！

```suggestion
        bitmapImageAsset.recycle()
```

Reply:
こちらのコミットで修正しました！

739e832ff8430e43b13cff47796287727866bf8e